### PR TITLE
Added an array with names of riders and added functionality for print…

### PR DESCRIPTION
**Pull request by yuliya-kovalenko**
…ing the array of riders if 'riders' argument is passed from command line

### CODE REVIEW

File: Motogp.java [hunk 1]
```diff
@@ -4,17 +4,32 @@ public static void main(String[] args) {
...
 		if (shouldPrintUpperCase) {
 			System.out.println("motogp".toUpperCase());
 		} else {
 			System.out.println("motogp");
 		}
+
+		/* I want a riders command so I can pass riders and it
+		will print out some of your favorite motogp riders.
+		1. create an array with names of riders
+		2. so if "rides" argument passed from command line > program prints an array*/
```
Comment (bobo333) [inline Motogp.java]: "riders" argument

**Comment by yuliya-kovalenko** [comment]
https://github.com/yuliya-kovalenko/motogp/issues/4

**Comment by yuliya-kovalenko** [comment]
Added 2 methods to handle "-u" command and "riders" command separately

**Comment by yuliya-kovalenko** [comment]
Used random to select randomly a race track from the array

**Pull request merged**